benchmark/run.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const statistics = require('./statistics');
+
+const yargsParser = require('yargs-parser');
+
+const args = yargsParser(
+  process.argv.slice(2),
+  {
+    alias: {
+      workers: ['W'],
+      connections: ['C'],
+      requests: ['R'],
+      size: ['S'],
+    },
+  }
+);
+
+const {
+  workers: workersAmount,
+  connections: connectionsPerWorker,
+  requests: requestsPerConnection,
+  size: argumentSize,
+} = args;
+
+
+const server = cp.fork(
+  path.join(__dirname, 'server'),
+  [workersAmount * connectionsPerWorker],
+  { stdin: 'pipe' }
+);
+
+let serverExited = false;
+
+const workers = new Array(workersAmount);
+const workersExited = new Array(workersAmount);
+
+const results = new Array(workersAmount);
```

Comment:
I don't care much, but it feels like this line and the previous one should have been swapped.